autoload/cmake4vim.vim

```diff
@@ -161,4 +161,17 @@ function! cmake4vim#SelectBuildType(buildType) abort
 
     silent call cmake4vim#GenerateCMake()
 endfunction
+
+function! cmake4vim#RunTarget() abort
+    if !exists('g:cmake_build_target')
+        echom 'Please select target first!'
+        return
+    endif
+    let s:exec_path = findfile(g:cmake_build_target, utils#cmake#getBuildDir())
```

Comment:
It works only if user don't change `CMAKE_RUNTIME_OUTPUT_DIRECTORY`. Need to highlight this limitation.

Reply:
Added comment in docs, that binary locations out of build directory are not supported.